js/services/ShareRoleCheck.js

```diff
@@ -0,0 +1,32 @@
+define(function (require, exports) {
+    var $ = require('jquery');
+    var constants = require('const');
+    const httpService = require('services/http');
+    
+    async function getRoleUsers(subject, permissionManagementRoleId) {
+	return await httpService.doGet(constants.apiPaths.roleUsers(permissionManagementRoleId))
```

Comment:
this call is failing on my side with 403, which I think makes sense. I would argue that the ability to query which users have any specific role, is an "admin level" type of query and should not be open to all. Can we change this code to just check if the current user has the role?
Or perhaps base the ability to share on a specific (new?) permission. Permissions can be accessed at `/user/me` endpoint.

Reply:
Or use the `user/{userId}/roles` endpoint:
https://github.com/OHDSI/WebAPI/blob/6a43da50ab2bf10df18306df658933ec6a49235e/src/main/java/org/ohdsi/webapi/service/UserService.java#L135-L138

Reply:
I tried this endpoint above here: https://github.com/uc-cdis/Atlas/pull/46/files 
But that also fails with 403. So, we'll need something else. I'm considering a new webapi endpoint as well as a possible solution if we don't want to introduce a specific permission for this share functionality (see my comments in my own PR)